assets/prefabs/vehicles/cargoCart.prefab

```diff
@@ -0,0 +1,56 @@
+{
+     "Location": {
+         "scale": 0.32
+     },
+     "Mesh": {
+         "mesh": "Rails:minecart",
+         "material": "Rails:minecart"
+     },
+     "RigidBody": {
+         "mass": 250,
+         "collisionGroup": "engine:world",
+         "collidesWith": [
+             "engine:world",
+             "engine:debris",
+             "engine:character",
+             "engine:default"
+         ]
+     },
+     "HullShape": {
+         "sourceMesh" : "Rails:minecart_bounds"
+     },
+     "CartJoint":{
+         "front": {
+             "range": 0.7
+         },
+         "back": {
+             "range": 0.7
+         }
+     },
+     "Health": {
+         "currentHealth": 1,
+         "maxHealth": 1,
+         "regenRate": 1,
+         "waitBeforeRegen": 2,
+         "destroyEntityOnNoHealth" : true
+     },
+     "AdditionalRails:CargoCart": {},
+     "Rails:RailVehicle": {},
+     "Trigger": {
+         "detectGroups": [
+             "engine:world",
+             "engine:debris",
+             "engine:sensor",
+             "engine:kinematic",
+             "engine:default"
+         ]
+     },
+     "Inventory": {
+         "privateToOwner": false,
+         "itemSlots": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
+     },
+     "InteractionTarget": {},
+     "InteractionScreen": {
+         "screen": "AdditionalRails:cargoCartScreen"
```

Comment:
Why can't you use the current inventory screen seeing as they appear to be basically the same?

Reply:
Ah fair enough. I believe it should be possible, even at the least to re-use some parts of it.